core/src/main/java/org/mapfish/print/servlet/job/HibernateAccounting.java

```diff
@@ -75,14 +76,18 @@ private void insertRecord(final HibernateAccountingEntry tuple) {
         tmpl.execute(
             new TransactionCallbackWithoutResult() {
               @Override
-              protected void doInTransactionWithoutResult(final TransactionStatus status) {
+              protected void doInTransactionWithoutResult(@Nonnull final TransactionStatus status) {
                 final Session currentSession = HibernateAccounting.this.sf.getCurrentSession();
                 currentSession.merge(tuple);
                 currentSession.flush();
                 currentSession.evict(tuple);
               }
             });
       } catch (HibernateException ex) {
+        String name =
+            MetricRegistry.name(
+                getClass().getSimpleName(), "insertRecordFailedWithHiddenHibernateException");
```

Comment:
Hidden?
It's in the logs!

Reply:
I think that the following one are more representative: `insertRecordFailedSkiped`, or `insertRecordFailedThenSkiped`, not?